src/qt/walletmodel.h

```diff
@@ -261,6 +262,9 @@ class WalletModel : public QObject
     // Watch-only address added
     void notifyWatchonlyChanged(bool fHaveWatchonly);
 
+    // Signal that wallet is about to be removed
+    void unload();
```

Comment:
Is this `unload()` method doing anything?

Reply:
oops. Actually I think this declaration is required for `BitcoinApplication` to connect the `removeWallet` slot to the `unload` signal (https://github.com/bitcoin/bitcoin/pull/13111/files#diff-8c9d79ba40bf702f01685008550ac100R471). Is that right?